debian/rules

```diff
@@ -1,12 +1,9 @@
 #!/usr/bin/make -f
 
-# export DH_VERBOSE=1
-
-DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+#export DH_VERBOSE=1
 
 %:
-	dh_update_autotools_config
```

Comment:
The `dh_update_autotools_config` target is already automatically executed by default:

```
$ debian/rules clean
$ dh binary --no-act
   dh_testdir
   dh_update_autotools_config 👈🏻
   ...
   ```